data-syncs/supported-data-sync-sources/rest-api.md

```diff
@@ -32,7 +32,15 @@ The following table outlines the mandatory and optional parameters you will find
+|       Parameter        |                                                                                               Description                                                                                                |                                              Example                                              |
+| ---------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------- |
+| Source                 | Mandatory. Select your source from the drop down menu.                                                                                                                                                   | REST API                                                                                          |
+| HTTP Method            | Mandatory.                                                                                                                                                                                               | This will be either GET or POST.                                                                  |
+| API Response Format    | Mandatory. Use this field to specify a response format of the endpoint. Currently, the Connections UI only supports JSON responses.                                                                      | JSON                                                                                              |
+| Records Root JSONPath  | Mandatory. Specify the JSON path for the results. If the JSON response top-element is an object, use `$.data` to return the parent object of the response, which will contain the array. | `$.data`, `$`                                                                                     |
+| Path to Iterate        | The path of the parameter to iterate. Use to flatten JSON objects.                                                                                                                                       |                                                                                                   |
```

Comment:
I don't think "flattening" makes the mental model required here obvious.

It could just be me, but I'd explain  (in too many words) it like "The JSONPath that selects the array of records of interest in the JSON document. A record is emitted for each element in that array which acts as the input to the source schema."